lib/child_process.js

```diff
@@ -136,7 +136,14 @@ function normalizeExecArgs(command, options, callback) {
   }
 
   // Make a shallow copy so we don't clobber the user's options object.
-  options = Object.assign({}, options);
+  options = Object.assign(Object.create(null), options);
+
+  // if (execOptions.hasOwnProperty('shell') &&
+  //   typeof execOptions.shell === 'string') {
+  //   execOptions.shell = options.shell;
+  // } else {
+  //   execOptions.shell = true;
+  // }
```

Comment:
Should delete the commented out code

Reply:
my bad, I pushed a commit to remove them